### DISCUSSION TITLE
Récompenses alternatives à l'xp de guilde pour les guildes level 100

### DISCUSSION THREAD

**Discussion by Thesmokingllama** [Crownicles/Crownicles#778] [Idées acceptées / déjà implémentées]
Permettre aux membres des guildes level 100 une récompense alternative (en argent ou en xp) à l'xp de guilde dans les mini-event et dans les transactions de pets.

**Comment by BastLast** [comment]
sooooon tm

**Comment by BastLast** [comment]
#1882 pour suivre le dev de cette suggestion ;)
